src/components/TaskList.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { fetchData, sortDateTime } from "./../utils/utils";
+import AddTaskButton from "./../components/AddTaskButton";
+import {
+	MainCont,
+	TaskBox,
+	DateText,
+	Description,
+	TaskTitle,
+	TaskCont,
+	CheckBox,
+	Icon,
+} from "../styled-components/reusables";
+
+import tick from "./../images/tick.svg";
+
+export default function TaskList({ url }) {
+	const [tasks, setTasks] = useState(null);
+
+	useEffect(() => {
+		resolveTaskData();
+	}, []);
+
+	function resolveTaskData() {
+		fetchData(`${url}`)
+			.then((result) => setTasks(result))
+			.catch((error) => error);
+	}
+
+	return (
+		<>
+			<AddTaskButton />
+			<MainCont width="80vw" paddingTop="5rem">
```

Comment:
Don't use abbreviations in naming, unless you have a good reason to (eg. long name that can't be changed). 
Also - I wonder why some of the CSS is in the styled-components and some inline?